lib/modules/dosomething/dosomething_api/resources/signup_resource.inc

```diff
@@ -189,7 +191,93 @@ function _signup_resource_index($user, $users, $campaigns, $count, $page, $compe
     'runs' => $runs,
   ];
 
-  return (new SignupTransformer)->index($parameters);
+  if (variable_get('rogue_collection', FALSE)) {
+    if ($user) {
+      $northstar_user_id = dosomething_user_get_northstar_id($user);
+      $user_object = user_load($user);
+    }
+
+     $rogue = dosomething_rogue_client();
+
+     $response = $rogue->getActivity([
+       'filter' => [
+         'northstar_id' => $northstar_user_id,
+         'campaign_id' => $campaigns,
+       ],
+     ]);
+
+    if ($response) {
+      $transformed_response = collect($response['data'])->map(function($signup) {
+        $campaign = node_load($signup['campaign_id']);
+        $campaign = Campaign::get($signup['campaign_id']);
+
+        if ($signup['posts']['data']) {
+          $posts = [];
+          $flagged = FALSE;
+
+          foreach ($signup['posts']['data'] as $post) {
+            $formatted_post = [];
+            $formatted_post['id'] = $post['id'];
+            $formatted_post['status'] = $post['status'];
+            $formatted_post['caption'] = $post['media']['caption'];
+            $formatted_post['uri'] = null;
+            $formatted_post['media']['uri'] = $post['media']['url'];
+            $formatted_post['media']['type'] = 'image';
+            $formatted_post['created_at'] = (new Carbon($post['created_at']))->timestamp;
+            // $formatted_post['kudos'] =
```

Comment:
@DFurnes Rogue doesn't return this information about kudos/reactions right now. Is this needed? If so, I'll add this to the response in Rogue's `/activity` endpoint!